### ISSUE TITLE
Does "connectWith" Even Work?

### ISSUE THREAD

**Issue by Spiderkeg** [mjsarfatti/nestedSortable#87]
I created two very simple ordered lists, both with unique IDs and same class.  I applied the connectWith option, using the same class.

While I can move an item from one list to another, the nesting function stops working at that point.  If the connectWith options is commented out, the nesting functionality begins working again.

Is this a known problem or is there something very specific I should be applying?  Any ideas?

**Comment by nback5** [comment]
This proves it works

http://jsfiddle.net/jhogervorst/Ge7eK/9/

**Comment by Spiderkeg** [comment]
Yes, yes it does, with jQuery 1.7.2 and jQueryUI 1.8.18.  I just tried your link with jQuery 1.9.1 and jQueryUI 1.9.2 and it didn't work.  I also wrote up a scenario where Sortables and nestedSortables were working together, with root parents and nestedsortable children, of which the newer jQuery doesn't work but the older 1.7.2 jQuery does work.  So you are half-correct... it does work with the older version.

Older: http://jsfiddle.net/Ge7eK/130/
Newer: http://jsfiddle.net/Ge7eK/131/

Older: http://jsfiddle.net/Ge7eK/132/
Newer: http://jsfiddle.net/Ge7eK/133/

**Comment by Mikeyheu** [comment]
Have the exact issue as well. My other plugins require newer versions of jQuery and jQuery UI.
